### PR TITLE
ARC-000: installing axios and react-query

### PR DESCRIPTION
**Pull request by chesterkudos**
## Summary
- creating basic form (no validation yet)
- setting up react-query client
- using hardcoded email/password
- poc for calling login endpoint
- adding env variables for base url
- added vite config option to enhance emotion classname output
- renaming component files to match component name
- adding git setup markdown file

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const sharedRules = {
   "no-mixed-spaces-and-tabs": "error",
   "no-nested-ternary": "error",
   "no-unneeded-ternary": "error",
+  "react-hooks/rules-of-hooks": "off",
```
Comment (jessie-kudos) [inline .eslintrc.js:11]: We do not usually have this turned off. Is there a good reason to do so?

Comment (jessie-kudos) [inline .eslintrc.js:11]: We don't usually turn this off. Is there a good reason to?

**Comment by jessie-kudos** [comment]
When is your plan to introduce MUI? Are we going to be using theming from there?

File: .eslintrc.js (hunk 1 above)
Comment (chesterkudos) [reply .eslintrc.js:11]: It was annoying so I turned it off, since this is a PoC, I will turn it back on though.

**Comment by chesterkudos** [comment]
MUI is coming in the next PR

**Pull request merged**